GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -1483,6 +1486,8 @@ bool IsCommitMessageValid()
                         try
                         {
                             if (!Regex.IsMatch(Message.Text, AppSettings.CommitValidationRegEx) &&
+                                !Message.Text.StartsWith(fixupPrefix) &&
+                                !Message.Text.StartsWith(squashPrefix) &&
```

Comment:
[nit] I'd put `!Message.Text.StartsWith(...)` before `!Regex.IsMatch(..)`, as I think the latter would be a more costly operation.
Just a gut feeling, haven't run any benchmarks

Reply:
Your message made me think again more carefully about it and indeed, you're right, so I have changed it.